test/api/walletFromEntropy/index.ts

```diff
@@ -0,0 +1,54 @@
+import assert from 'assert-diff'
```

Comment:
All the wallet creation methods should be in one file under `test/wallet` or something, they don't need to be under `api` (which is intended to just be the rippled-connected stuff). You could put them in one file.